### ISSUE TITLE
Re-Training U-Net has poor accuracy

### ISSUE THREAD

**Issue by adib** [plaidml/plaidml#218]
When I re-train [U-Net](https://github.com/zhixuhao/unet) using the PlaidML backend with AMD/OpenCL driver, the loss value stays firmly above 1.x and did not improve after five iterations. 

In contrast, using the Tensorflow CPU backend, the first epoch already results in a loss values below 1 and improves on each subsequent epoch.

PlaidML version: 0.3.5

**Comment by brianretford** [comment]
Could you try and reproduce with the latest version? If it does, could you provide a more precise reproduction (dataset set you're retraining on, etc). We'll have instructions soon on using our backend tests to help us debug. 

Apologies for the delay.

**Comment by dgkutnic** [comment]
Issues in the `needs info` state for 7+ days will be marked as obsolete and closed. If the issue is still occurring for you, please re-open the issue and provide us with the requested info.
